sts_auth/okta.py

```diff
@@ -25,41 +25,38 @@ def __init__(self, session, state_token, okta_org, okta_shared_secret=None):
         self.okta_shared_secret = okta_shared_secret
 
     def fetch_available_mfa_factors(self):
-        okta_auth_url = 'https://{}.okta.com/api/v1/authn'.format(self.okta_org)
-        okta_transaction_state = self.session.post(
-            okta_auth_url,
-            json={'stateToken': self.state_token}
-        )
-        okta_factors = okta_transaction_state.json().get('_embedded', {}).get('factors', {})
+        okta_auth_url = "https://{}.okta.com/api/v1/authn".format(self.okta_org)
+        okta_transaction_state = self.session.post(okta_auth_url, json={"stateToken": self.state_token})
+        okta_factors = okta_transaction_state.json().get("_embedded", {}).get("factors", {})
         if len(okta_factors) > 0:
             # Format the factors as {factorType: {details..}, ..}
             # so it will be easier to pick pull out by type later
-            return {f['factorType']: f for f in okta_factors}
+            return {f["factorType"]: f for f in okta_factors}
         else:
-            msg = ('No Okta MFA methods available.\n'
-                   'Please visit https://{}.okta.com to configure Okta MFA.')
-            click.secho(msg.format(self.okta_org), fg='red')
+            msg = "No Okta MFA methods available.\n" "Please visit https://{}.okta.com to configure Okta MFA."
+            click.secho(msg.format(self.okta_org), fg="red")
             sys.exit(1)
 
     def handle_okta_verification(self, response):
         # If there is no assertion, and we find an Okta portal on the page,
         # we have to pass through the Okta portal regardless of whether MFA
         # will be required or not.
         if not self.okta_org:
-            msg = ('Okta MFA required but no Okta Organization set. '
-                   'Please either set in the config or use `--okta-org`')
-            click.secho(msg, fg='red')
+            msg = (
+                "Okta MFA required but no Okta Organization set. " "Please either set in the config or use `--okta-org`"
```

Comment:
Changes like these should be merged into one string if they're moved to a single.

Reply:
@phillipjf fixed